components/wmtk_components/adaptive_tessellation/src/wmtk/components/adaptive_tessellation/image/load_image_exr.cpp

```diff
@@ -325,12 +324,13 @@ auto load_image_exr_split_3channels(const std::filesystem::path& path) -> std::
 
     wmtk::logger().debug("[load_image_exr_3channels] done \"{}\"", path.string());
 
-    return {static_cast<size_t>(exr_image.width),
-            static_cast<size_t>(exr_image.height),
-            std::move(index_red),
-            std::move(index_green),
-            std::move(index_blue),
-            std::move(data_r),
-            std::move(data_g),
-            std::move(data_b)};
+    return {
+        static_cast<size_t>(exr_image.width),
+        static_cast<size_t>(exr_image.height),
+        std::move(index_red),
```

Comment:
lol i know you didn't write this but std::move on primitives doesn't do anything but potentially break optimizers